.travis.yml

```diff
@@ -5,7 +5,7 @@ branches:
   only:
     - master
 before_install:
-  - wget https://repo.continuum.io/miniconda/Miniconda3-4.3.30-Linux-x86_64.sh
+  - wget https://repo.continuum.io/miniconda/Miniconda3-4.3.31-Linux-x86_64.sh
```

Comment:
I'm unclear on how the Miniconda versioning relates to the conda versioning.  Does this mean that we're still using conda 4.3 for Travis because 4.4 isn't available in a Miniconda installer yet?  I see that `Miniconda3-4.3.31-Linux-x86_64.sh` is the most recent installer.

Can we update conda to 4.4 after installing Miniconda so that our Travis install matches the readme and the new `conda activate manubot` syntax?

Reply:
Yeah I think you're understanding is correct. It'd be nice to have travis use `conda activate manubot`. However, my preference is to do this in the future, when a Miniconda is released that contains conda 4.4.